README.md

```diff
@@ -58,9 +58,9 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v3
     - name: set up JDK 1.8
-      uses: actions/setup-java@v1
+      uses: actions/setup-java@v3
       with:
         java-version: 1.8
```

Comment:
`setup-java` requires setting a `distribution` argument.
```suggestion
      uses: actions/setup-java@v3
      with:
        distribution: 'zulu'
        java-version: 1.8
```
\+ I personally, would also bump java version to the latest LTS 🤷, so:
```suggestion
      uses: actions/setup-java@v3
      with:
        distribution: 'zulu'
        java-version: 17
```
but I'm not sure what @wzieba's thought are here 😉